### PR TITLE
Register previously added metrics in ghproxy

### PR DESCRIPTION
**Pull request by jmguzik**
Forgot to register a new metrics for ghproxy throttling algoritm

Signed-off-by: Jakub Guzik <jguzik@redhat.com>

### CODE REVIEW

File: ghproxy/ghmetrics/ghmetrics.go [hunk 1]
```diff
@@ -111,6 +111,7 @@ func init() {
 	prometheus.MustRegister(cacheCounter)
 	prometheus.MustRegister(timeoutDuration)
 	prometheus.MustRegister(cacheEntryAge)
+	prometheus.MustRegister(ghRequestWaitDurationHistVec)
```
Comment (petr-muller) [inline ghproxy/ghmetrics/ghmetrics.go:114]: nit: can we put this together with the other `ghRequest...` register?

/ok-to-test

Comment (jmguzik) [reply ghproxy/ghmetrics/ghmetrics.go]: Changed, also ordering in the file is changed (all gh metrics come first).

**Review by petr-muller — APPROVED** [review]

File: ghproxy/ghmetrics/ghmetrics.go [hunk 2]
```diff
@@ -58,6 +58,17 @@ var ghRequestDurationHistVec = prometheus.NewHistogramVec(
 	[]string{"token_hash", "path", "status", "user_agent"},
 )
 
+// ghRequestDurationHistVec provides the 'github_request_duration' histogram that keeps track
+// of the duration of GitHub requests by API path.
+var ghRequestWaitDurationHistVec = prometheus.NewHistogramVec(
+	prometheus.HistogramOpts{
+		Name:    "github_request_wait_duration_s",
```
[the remark this answers is not in the document]
Comment (jmguzik) [reply ghproxy/ghmetrics/ghmetrics.go]: done

File: ghproxy/ghmetrics/ghmetrics.go [hunk 3]
```diff
@@ -58,6 +58,17 @@ var ghRequestDurationHistVec = prometheus.NewHistogramVec(
 	[]string{"token_hash", "path", "status", "user_agent"},
 )
 
+// ghRequestDurationHistVec provides the 'github_request_duration' histogram that keeps track
+// of the duration of GitHub requests by API path.
+var ghRequestWaitDurationHistVec = prometheus.NewHistogramVec(
+	prometheus.HistogramOpts{
+		Name:    "github_request_wait_duration_seconds",
+		Help:    "GitHub request wait duration before sending to API in s",
```
Comment (alvaroaleman) [inline ghproxy/ghmetrics/ghmetrics.go:66]:
```suggestion
		Help:    "GitHub request wait duration before sending to API in seconds",
```

Comment (jmguzik) [reply ghproxy/ghmetrics/ghmetrics.go]: Done

**Review by alvaroaleman — APPROVED** [review]

**Pull request merged**